src/test/java/SalesforceModules/QuoteUtil.java

```diff
@@ -21,8 +23,7 @@ public class QuoteUtil {
      *
      * @param logStep the log
      */
-    public static void NewButtonQuote(String logStep) throws InterruptedException {
-        Thread.sleep(5000);
+    public static void NewButtonQuote(String logStep){
```

Comment:
function name should start from small letter.

Reply:
Updated as per comments